### PR TITLE
Fix double click event listener and disable on action component

### PR DESCRIPTION
**Pull request by ptychu**
Fix for #12 .
When action is clicked listener is executed even if action is supposed
to be disabled. Additionally when action is enabled clicked listener is
executed twice.
Due to this akward behaviour strange null check was required in
updateProfile() listener in profile.component.

### CODE REVIEW

**Comment by ptychu** [comment]
Hi @lasfrancisco, can you take a look on this changes? I'd add you as a reviewer, but I don't have any permissions ;)

**Comment by lasfrancisco** [comment]
I do like your approach. You just need to review the way the disabled input works. I'd recommend adding the coerceBooleanValue() logic to the input (take a look at the way it is done in elements) then use the internal disabled boolean property in the host property of the component decorator like `host: { '[style.pointer-events.none]': 'disabled' } for the PR to get a green light :)

File: apps/wizdm/src/app/navigator/actionbar/action.component.ts [hunk 1]
```diff
@@ -23,9 +23,14 @@ export class ActionComponent {
...
   public disabled = false;
 
-  /** Emits when clicked */
-  @Output() activate = new EventEmitter<MouseEvent>();
+  @HostBinding('style.pointer-events') get isActionDisabled(): string { return this.disabled ? 'none' : ''; }
+
+  public onClick(event: MouseEvent) {
+
+    // Notifies teh actionbar about the action that has been activated
+    this.actionbar.activate(this);
```
Comment (ptychu) [inline apps/wizdm/src/app/navigator/actionbar/action.component.ts:34]: @lasfrancisco I've restored `actionbar.activate()`. I'm not sure if it adds any value though... If it's not needed button listener could be removed all together...

File: apps/wizdm/src/app/navigator/actionbar/action.component.ts [hunk 2]
```diff
@@ -23,9 +23,14 @@ export class ActionComponent {
   @Input() color: ThemePalette;
 
   /** Disables the action */
-  @Input('disabled') set disableAction(value: boolean) { this.disabled = coerceBooleanProperty(value); }
+  @Input('disabled') set disableAction(value: any) { this.disabled = coerceBooleanProperty(value); }
```
Comment (ptychu) [inline apps/wizdm/src/app/navigator/actionbar/action.component.ts:26]: I changed type to any, as it better reflects what `coerceBooleanProperty` will do...

**Pull request merged**